tests/cases/fourslash/organizeImports6.ts

```diff
@@ -0,0 +1,11 @@
+/// <reference path="fourslash.ts" />
+
+// Regression test for GH#43107
+
+//// import * as something from "path"; /* small comment */ // single line one.

```

Comment:
I think you want this single line comment test to also not delete the import so you can see that the comment is preserved (by adding usages of `something` and `somethingElse` below).

Reply:
It was actually meant to test that the single line is deleted. I found I used to have that problem. It also makes sense to test the contrary so I have updated to do it for both.